### PR TITLE
Generate sitemap.xlm and robots.txt

### PR DESCRIPTION
**Pull request by natapg**
Added sitemap.xml and robots.txt to our website using Gatsby plugins
Closes WEB-70

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -13,10 +13,13 @@ module.exports = {
   siteMetadata: {
     title: 'FingerprintJS Pro - browser fingerprinting & fraud detection API',
     description: 'Stop fraud, spam, and account takeovers with 99.5% accurate browser fingerprinting as a service.',
+    siteUrl: 'https://fingerprintjs.com',
```
Comment (Lisenish) [inline gatsby-config.js:16]: You probably can't use TS file here, I get it, but you can still move this string to the constant variable in this file

**Comment by Lisenish** [comment]
Pls delete `package-lock.json` file, we use yarn

**Review by Lisenish — CHANGES_REQUESTED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (natapg) [reply gatsby-config.js:16]: you're right, I'm going to make the change

Comment (natapg) [reply gatsby-config.js:16]: you're right, I'm going to move it to a constant variable

File: gatsby-config.js [hunk 2]
```diff
@@ -13,10 +14,13 @@ module.exports = {
   siteMetadata: {
     title: 'FingerprintJS Pro - browser fingerprinting & fraud detection API',
     description: 'Stop fraud, spam, and account takeovers with 99.5% accurate browser fingerprinting as a service.',
-    url: 'https://fingerprintjs.com',
+    siteUrl: baseUrl,
+    url: baseUrl,
```
Comment (Lisenish) [inline gatsby-config.js:18]: @tan-90 AFAIK It's common to have it called stiteUrl in metadata and some plugins rely on that convention, so we need to refactor url usages to use siteUrl intead (in some separate task)

**Review by Lisenish — APPROVED** [review]

**Pull request merged**